modules/ROOT/pages/portal-vanity-domain.adoc

```diff
@@ -0,0 +1,105 @@
+= Public Portal Vanity Domain
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This repository contains instructions and the necessary NGINX configuration for setting up an organization's public portal using a vanity domain. It also provides a docker image for running this configuration.
```

Comment:
Per Docker's legal trademark guidelines, Docker should be capitalized. "Our Marks should be written with the proper capitalization as shown in these guidelines. For example: “Docker”. Docker may be written in lower case as ‘docker’ only when the use explicitly refers to command line commands. For example in documentation, help text or examples of use." 
I only know this because I was working on Docker at my last gig and I got into a big fight with engineers over it.  haha